libs/ledger-live-common/src/__tests__/test-helpers/bridgeSetupUpdateMocks.ts

```diff
@@ -0,0 +1,53 @@
+import { PerformanceObserver, PerformanceObserverCallback } from "node:perf_hooks";
+import {
+  HttpPerformanceEntry,
+  HttpPerformanceEntryRequest,
+  HttpPerformanceEntryResponse,
+  StdRequest,
+} from "./types";
+import { createHash } from "crypto";
+
+function sha256(str: string) {
+  return createHash("sha256").update(str).digest("hex");
+}
+
+/* 
+Maps objects returned by node:perf_hooks into our model StdRequest
+*/
+function reqToStdRequest(
+  nodeRequest: HttpPerformanceEntryRequest,
+  nodeResponse: HttpPerformanceEntryResponse,
+): StdRequest {
+  return {
+    method: nodeRequest.method,
+    url: nodeRequest.url,
+    headers: nodeRequest.headers,
+    body: nodeRequest.body != null ? nodeRequest.body : {},
+    fileName: sha256(`${nodeRequest.method}${nodeRequest.url}`),
```

Comment:
maybe we can exclude the hostname of nodeRequest.url because we may change the node from time to time but we want to keep the filename I think.